rudder-core/src/main/scala/com/normation/rudder/services/policies/SystemVariableService.scala

```diff
@@ -353,6 +353,8 @@ class SystemVariableServiceImpl(
       //IT IS VERY IMPORTANT TO SORT SYSTEM VARIABLE HERE: see ticket #4859
       val varManagedNodesAdmin = systemVariableSpecService.get("MANAGED_NODES_ADMIN").toVariable(children.map(_.localAdministratorAccountName).distinct.sorted)
 
+      //IT IS VERY IMPORTANT TO SORT SYSTEM VARIABLE HERE: see ticket #4859
+      val varManagedNodesIp = systemVariableSpecService.get("").toVariable(children.flatMap(_.ips).distinct.sorted)
```

Comment:
no, we discussed that there should be a config somewhere to say "not secure" and use authorized network in place. 

But you're right, I'm not sure where the logic for that should go. It does not seems right to put it in the variable, else it's name will be misleading a lot. What do you think ?